### DISCUSSION TITLE
从 skyline 到 OpenGalaxy

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#61] [2️⃣ 聊项目]
今天群里又有人提到了 GitHub 的 skyline 应用，主要还是不满意，觉得做的粗糙，但又没有更好的展示开发者贡献的元宇宙应用了，我们其实可以想想哈，OpenGalaxy 前途无量~ :-)

https://skyline.github.com/

![image](https://user-images.githubusercontent.com/15010826/170905016-22bff522-6a73-4586-98bd-df8daa85e97f.png)

**Comment by will-ww** [comment]
发现还有实物~

![94da4d430c5af455b7f183757e2c81b](https://user-images.githubusercontent.com/15010826/171071983-9a9a6b2f-8308-48d3-8dcb-cab7f423db25.jpg)

**Comment by tyn1998** [comment]
实物精致多了哈哈

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Will Wang ***@***.***>
发送时间: Tuesday, May 31, 2022 8:45:29 AM
收件人: X-lab2017/open-wonderland ***@***.***>
抄送: Subscribed ***@***.***>
主题: Re: [X-lab2017/open-wonderland] 从 skyline 到 OpenGalaxy (Discussion #61)


发现还有实物~

[94da4d430c5af455b7f183757e2c81b]<https://user-images.githubusercontent.com/15010826/171071983-9a9a6b2f-8308-48d3-8dcb-cab7f423db25.jpg>

―
